core/unitOps.chainmail

```diff
@@ -1,7 +1,11 @@
 /**
- * Unitss are wrappers on extents that have been validated by an UnitOps, and
- * can be verified as having been issued by the UnitOps. They contain their
- * extent and a Label. The label identifies a particular assay.
+ * Units are asset descriptions and take the form of labeled extents. 
+ * UnitOps execute the logic of how units are changed when digital 
+ * assets are merged, separated, or otherwise manipulated. For
+ * instance, a deposit of 2 bucks into a purse that already has 3 bucks
```

Comment:
A pervasive issue is the use of "for instance". A minor issue, Since we are explaining to object programmers, I always use "for example" instead. When I see "instance", it always takes me an extra moment to think "oh, not that meaning of 'instance' ". Might be a rare reaction, but probably not unique to me.

Reply:
Sure, we can try to use "for example" more